homeassistant/components/reolink/manifest.json

```diff
@@ -18,5 +18,5 @@
   "documentation": "https://www.home-assistant.io/integrations/reolink",
   "iot_class": "local_push",
   "loggers": ["reolink_aio"],
-  "requirements": ["reolink-aio==0.7.12"]
+  "requirements": ["reolink-aio==0.7.13"]
```

Comment:
The py.typed file should normally be empty is my experience. Please remove the comment unless there's some documentation that says it's ok to add a comment.

Reply:
Mypy itself has a comment in its py.typed file that I used as example:
https://github.com/python/mypy/blob/master/mypy/py.typed

The problem was that I forgot to add the py.typed file to setup.py using `package_data={"reolink_aio": ["py.typed"]},`
So although it was uploaded to reolink-aio github, it was not part of the pypy package.

Reply:
Ok.